src/Components/Hamburger/Hamburger.jsx

```diff
@@ -0,0 +1,35 @@
+import "./Hamburger.css";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../context/auth-context";
+
+const Hamburger = () => {
+  const { logoutUser } = useAuth();
+
+  return (
+    <div className="hamburger flex-column pt-3">
+      <Link to="/" className="list_item flex-row p-3">
+        Home
+      </Link>
```

Comment:
1.this is a repeating line of code you can map an array with id, link,title as if you will increase options in hamburger in future 

2.also benefits you can also add Icon in menu

Reply:
Oh Yeah, I forgot . thanks For suggesting.. Will do it now.